pre_commit_mirror_maker/languages.py

```diff
@@ -14,10 +14,20 @@ def ruby_get_package_versions(package_name: str) -> list[str]:
     return list(reversed([version['number'] for version in resp]))
 
 
+def _node_get_package_versions(package_json):
+    versions = package_json['versions']
+    version_timestamps = package_json['time']
+
+    # Sort the versions according to their timestamps, to prevent versions
+    # from being missed if they get released out of order.
+    versions.sort(key=lambda e: version_timestamps.get(e, ''))
+    return versions
+
+
 def node_get_package_versions(package_name: str) -> list[str]:
     cmd = ('npm', 'view', package_name, '--json')
-    output = json.loads(subprocess.check_output(cmd))
-    return output['versions']
+    output = json.loads(subprocess.check_output(cmd, shell=True))
```

Comment:
The `shell=True` change was required for me to run the script on Windows. I can revert it if necessary.